src/js/editor/editor.js

```diff
@@ -319,6 +332,9 @@ class Editor {
       showForTag: 'a'
     }));
 
+    // A call to `run` will trigger the didUpdatePostCallbacks hooks with a
+    // postEditor.
+    this.run(() => {});
```

Comment:
Maybe this should be `this.runCallbacks()` or something? This looks like the sort of no-op-y code that will get deleted by someone in the future.